src/PHASE_3_RANSAC_FILTERING/ransac_filter.py

```diff
@@ -0,0 +1,190 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Oct 16 14:46:18 2018
+
+@author: mmacferrin
+
+Code using OpenCV, from "Learn OpenCV" tutorial: https://www.learnopencv.com/image-alignment-feature-based-using-opencv-c-python/
+"""
+
+#######################################################################################
+## Code for importing the Utilities directory
+import os,sys,inspect
+currentdir = os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe())))
+parentdir = os.path.dirname(currentdir)
+utilities_dir = os.path.join(parentdir, "UTILITIES")
+sys.path.insert(0,utilities_dir)
+#######################################################################################=
+
+
+from Matches_Reader import ASIFT_Matches
```

Comment:
capitalization issue, file names in the directory are lowercase while here it is uppercase not matching the file names.

Reply:
You're right, I'm fixing the capitalization issue in a few of the files, as well as the capitalization of the PHASE_... directories (turn them to lowercase "phase_...."). I can either re-commit those changes to my branch. If I do, would they make it into the pull request? (I'm guessing not.) If there's a best-case workflow for me to make those changes, just let me know.

Reply:
You can push new changes to your branch and they will be included in the pull request.

Reply:
yes, we will keep reviewing the pull request until all the issues are solved then we can move forward and accept it to be merged with devel branch.

Reply:
Thumbs up. Just committed some of the small fixes, although admittedly I didn't fully test all the code to ensure bugs didn't creep in. We have a telecon in 5 minutes, I'll test and make bug fixes as needed.

(Aymen caught one with the command-line interface in the ransac_filter.py code, for instance.)

Reply:
Thanks, @mmacferrin for the fast responses.